Tasks/Docker/container.ts

```diff
@@ -21,38 +21,28 @@ else {
     authenticationProvider = new GenericAuthenticationTokenProvider(tl.getInput("dockerRegistryEndpoint"));
 }
 
-authenticationProvider.getAuthenticationToken().then(
-    function success(registryAuthenticationToken) {
```

Comment:
Now Authentication Provider is not returning promise its returning authentication token directly, since rest call is not required anymore. sucess and failure method only exists for the promise.